src/page-multi-facility/FacilityInputForm.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useContext, useState } from "react";
+import { useHistory } from "react-router-dom";
+import styled from "styled-components";
+
+import { saveFacility } from "../database/index";
+import InputButton from "../design-system/InputButton";
+import InputText from "../design-system/InputText";
+import ChartArea from "../impact-dashboard/ChartArea";
+import ImpactProjectionTable from "../impact-dashboard/ImpactProjectionTableContainer";
+import MitigationInformation from "../impact-dashboard/MitigationInformation";
+import useModel from "../impact-dashboard/useModel";
+import { FacilityContext } from "./FacilityContext";
+import FacilityInformationSection from "./FacilityInformationSection";
+import LocaleInformationSection from "./LocaleInformationSection";
+
+const FacilityInputFormDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+const LeftColumn = styled.div`
+  margin: 20px;
+`;
+const RightColumn = styled.div`
+  margin: 20px;
+`;
+
+// TODO add section header tooltips
+// TODO add summary at bottom of Locale Information
+const FacilityInputForm: React.FC = () => {
+  const { facility } = useContext(FacilityContext);
+  const history = useHistory();
+  const [facilityName, setFacilityName] = useState(facility?.name);
+  const model = useModel();
+
+  const save = () => {
+    saveFacility({
```

Comment:
There may be a bit of a race condition here where the data first needs to be persisted before moving on to the next page or else you may see an error that says something along the lines of "toDate can't be set null".  I think this is because we load the `updatedAt` on the next page to show when the facility was updated and it's not set yet.  Not sure if there is a wait to make this synchronous by calling `await` here or not.

Of course, I can't reproduce the issue now so I don't know if it's still a problem but wanted to flag for you all

Reply:
We'll keep an eye out during the fit and finish pass, and assign the bug fix to someone if necessary!